packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/get.js

```diff
@@ -883,6 +883,26 @@ describe('plugin-conversation', function () {
             assert.isTrue(actCount < minActivities);
           }));
 
+        it('should return all activities in a space when jumping to an activity in a space with more activities than the asked-for activities limit', () => webex.internal.conversation.create({participants: [scott.id]})
+          .then((c) => {
+            const $posts = [];
+
+            // eslint-disable-next-line no-plusplus
+            for (let i = 0; i < 15; i++) {
+              $posts.push(webex.internal.conversation.post(c, {displayName: `message ${i}`}));
+            }
+
+            return Promise.all($posts);
+          })
+          .then((posts) => posts[posts.length - 1]).then((lastPost) => jumpToActivity(lastPost))
```

Comment:
Sorry I didn't think about this in my first review, if you want to make sure you get the very last post, sending one final post after your initial batch might make the test more resilient. We could get a promise resolution that takes longer to return for one of the non-last posts that could break the test.